aexpect/ops_linux.py

```diff
@@ -216,6 +216,29 @@ def is_nonzero_size_file(session, path):
 ###############################################################################
 
 
+def glob(session, glob_pattern):
+    """
+    Find files matching given pattern through a session.
+
+    :param session: session to run the command on
+    :type session: ShellSession
+    :param str glob_pattern: pattern of filenames where `*`, `?` and such match
+                             any string or not-empty-string and so on.
+    :returns: file names matching pattern, including path (like glob.glob)
+    :rtype: [str]
+
+    Much like :py:func:`glob.glob` but implemented using `echo`.
+    """
+    cmd = f'echo {glob_pattern}'
+    status, output = session.cmd_status_output(cmd)
+    status, output = _process_status_output(cmd, status, output)
+    paths = []
+    for path in output.split():
+        if test(session, path, "-e"):
```

Comment:
I liked this idea to ensure the path exists, but the cons is it's really slow. For 100 files it took 50s on my PC. Provided you fix the glob to really return the correct paths, this should not be needed, right?

Reply:
We could then just fall back to the original `ls` solution hiding the status. Even though I had to investigate for solutions for not hiding the status and we end up back where we started which sounds like some wasted time and effort. Or do you have a better solution you could propose?

Reply:
Looks like there is no perfect solution. There will always be some limitation. The `ls` solution should do in most cases s I think it makes sense to use that one and leave it up to people to use custom handling in case this fails.

Reply:
Alright, pushed the changes just now, I think CI will pass.